### PR TITLE
IE11 fixes

### PR DESCRIPTION
**Pull request by ocke**
IE 11 had 3 problems which are fixed in this PR.

- no Element.prototype.remove
- no CustomEvent
- some extra guard code around getting an element

cc: @zendesk/apps-migration

### CODE REVIEW

File: src/javascript/lib/helpers.js [hunk 1]
```diff
@@ -5,6 +5,13 @@ export function getElement (selector, el = document) {
   return el.querySelector(selector)
 }
 
+export function removeElement (el) {
+  if (typeof el === 'string') el = getElement(el)
+
+  // IE11 compatible
+  if (el.parentNode) el.parentNode.removeChild(el)
+}
```
Comment (keitheous) [inline src/javascript/lib/helpers.js]: 👍

File: src/javascript/lib/datalist.js [hunk 2]
```diff
@@ -229,11 +229,21 @@ export default class Datalist {
...
 
   trigger (eventName) {
-    const event = new window.CustomEvent(eventName, { detail: this })
+    const event = customEvent(eventName, { detail: this })
     this.$datalist().dispatchEvent(event)
   }
 }
 
+// IE11 compatible customEvent function
+function customEvent (event, params) {
```
Comment (hoangtrvu) [inline src/javascript/lib/datalist.js]: looks like this can be put in `helpers` too

File: src/javascript/lib/helpers.js [hunk 3]
```diff
@@ -253,3 +260,13 @@ export function intersection (group1, group2) {
 export function anyOverlap (group1, group2) {
   return !!intersection(group1, group2).length
 }
+
+// IE11 compatible customEvent function
+export function customEvent (event, params) {
+  if (typeof window.CustomEvent === 'function') return new window.CustomEvent(event, params)
+
+  params = params || { bubbles: false, cancelable: false, detail: undefined }
```
Comment (yu-tian113) [inline src/javascript/lib/helpers.js]: How about set default params in the function argument list

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+(Title field)
+[OFFAPPS-XXX] Short description
```
Comment (vaevictis) [inline .github/pull_request_template.md]: ❤️ for unifying PRs all around

**Pull request merged**